apps/i18n/common/ru_ru.json

```diff
@@ -99,52 +99,52 @@
   "assignCourse": "Назначить курс",
-  "assignConfirm": "Вы уверены, что вы хотите назначить «{assignmentName}» классу «{sectionName}»?",
-  "assignHiddenUnitConfirm": "Похоже, что вы ранее скрыли «{assignmentName}» из класса «{sectionName}». Вы все еще хотите назначить этот блок и сделать его видимым?",
+  "assignConfirm": "Вы уверены, что хотите назначить \"{assignmentName}\" для раздела \"{sectionName}\"?",
+  "assignHiddenUnitConfirm": "Похоже, что Вы ранее скрыли \"{assignmentName}\" из раздела \"{sectionName}\". Вы все еще хотите назначить этот блок и сделать его видимым?",
   "assigned": "Назначено",
   "assignedTo": "Назначено",
   "assignedToWithColon": "Назначено:",
-  "assignToSection": "Назначить классу",
+  "assignToSection": "Назначить для раздела",
   "assignUnit": "Назначить блок",
-  "assignedToNewerVersion": "Ваш учитель записал вас на новую версию данного курса.",
+  "assignedToNewerVersion": "Твой учитель записал тебя на новую версию этого курса.",
   "assignmentSelectorCourse": "Выберите курс",
   "assignmentSelectorVersion": "Версия",
   "assignmentSelectorUnit": "Выберите текущий блок",
   "atHomeBannerBody1": "Поддержка для родителей учеников и учителей закрывшихся школ. ",
-  "atHomeBannerBody2": "Посмотрите учебные ресурсы, которые мы предлагаем.",
+  "atHomeBannerBody2": "Посмотрите предлагаемые нами учебные ресурсы.",
   "atHomeBannerLinkText": "Начало работы",
-  "audioInitializeError": "Ваш компьютер не настроен для записи аудио. Пожалуйста, убедитесь, что у вас есть микрофон с разрешения для доступа к аудио на этом веб-сайте.",
+  "audioInitializeError": "Ваш компьютер не настроен для записи аудио. Пожалуйста, убедитесь, что у Вас есть микрофон с правом доступа к аудиофайлам на этом сайте.",
   "audioSaveError": "Произошла ошибка при сохранении записи. Пожалуйста, запишите повторно и повторите попытку. Если эта проблема не устраняется, пожалуйста, обновите страницу и повторите попытку.",
-  "authorizeGoogleClassrooms": "Нажмите здесь для авторизации в Google классе.",
-  "authorizeGoogleClassroomsText": "Для того, чтобы синхронизировать с Google класс, вы должны дать Code.org разрешение на доступ к вашей учетной записи на Google Classroom.",
+  "authorizeGoogleClassrooms": "Нажмите здесь, чтобы разрешить доступ к Google-классу.",
+  "authorizeGoogleClassroomsText": "Для синхронизации данных с Google-классом Code.org должен иметь действительный доступ к Вашей учетной записи на Google Классах.",
   "authorName": "**Автор:** {name}",
   "autoGenerated": "Автоматически сгенерированные",
-  "autolock": "Примечание: Этап автоматически заблокируется через 24 часа.",
-  "availableLessons": " Доступно в {numLessons, plural, one {1 Уроке} other {# Уроках}}:",
+  "autolock": "Примечание: этот этап будет автоматически заблокирован через 24 часа.",
+  "availableLessons": " Доступно в {numLessons, plural, one {1 Lesson} других {# Lessons}}:",
```

Comment:
FYI @castro-jorge that TBH appears to have translated some syntax here, breaking this string https://crowdin.com/translate/codeorg/43/enus-ru#2290691

Reply:
https://crowdin.com/project/codeorg/activity_stream?translation_id=2290691&lang=41

Reply:
Looks like they actually broke a number of strings:
- https://crowdin.com/project/codeorg/activity_stream?translation_id=88706&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=404240&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=110733&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=1788958&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=110735&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=2373026&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=2373028&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=112104&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=112105&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=79096&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=3095&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=137005&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=137006&lang=41
- https://crowdin.com/project/codeorg/activity_stream?translation_id=136978&lang=41